google-cloud-spanner/pom.xml

```diff
@@ -377,5 +377,38 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
This was a missing piece of configuration that was still in the JDBC repository, but that belongs in the Connection API. It is a Maven profile for automatically generating tests for new client side SQL statements.

Reply:
Could we add this as a separate PR? Otherwise we won't have this until the commit stats is merged into master.

Reply:
Good point, I've added #611 for that, and I'll remove it from this PR.